pkg/plugins/resources/dockerimage/spec.go

```diff
@@ -104,7 +104,7 @@ func NewDockerImageSpecFromImage(image, tag string, auths map[string]docker.Inli
 	return &dockerimagespec
 }
 
-// NewFrilterFromValue tries to identify the closest tagFilter based on an existing tag
+// NewFilterFromValue tries to identify the closest tagFilter based on an existing tag
```

Comment:
```suggestion
// getTagFilterFromValue tries to identify the closest tagFilter based on an existing tag
```

This one is even worse than a typo. It should be the function's (exact) name